src/test/sigencoding_tests.cpp

```diff
@@ -0,0 +1,389 @@
+// Copyright (c) 2018 The Bitcoin developers
```

Comment:
need to add `sigencoding_tests.cpp` to `.formatted-files`

Reply:
Done. Also added the other missing files. See bottom "Cosmetics" commit.